content/en/api/payments.md

```diff
@@ -5,4 +5,99 @@ position: 41
 category: API
 ---
 
-Payments
\ No newline at end of file
+## Capture
+
+The capture endpoint allows capturing funds from a Single Payment authorisation, a Frequent Payment or a Frequent Payment authorisation. It also allows getting the details of a previous capture.
+
+### Capture payment
+
+`POST` `/capture/<payment id>`
+
+#### Parameters
+
+<div class="ep-protocol-list">
+
+  - `transaction_key`: `string` (<= 50 characters) with the internal key used to identify this transaction in the merchant's database.
+  - `capture_date`: `string` in the format `'Y-m-d'` with the capture date.
+  - `account`: *optional* `Object` for multi-account clients.
+    - `id`: `string` with the uuid of the account.
+  - `splits`: *optional* `Array` of `Object`s used in split payments.
+    <details><summary>Expand</summary>
+      
+      Each `Object` has the following properties:
+    - `split_key`: `string` (<= 50 characters) with the merchant's internal key for identifying the split.
+    - `split_descriptive`: `string` (<= 255 characters) with the description of the split.
+    - `value`: `number` the split funds, rounded to 2 decimals. (*required*)
```

Comment:
I would put the _`*required*`_ as the _`*optional*`_ before the type

Reply:
I had it like that but visually in the generated document (or app, in this case) I preferred having them at the end. Right now it's at the end everywhere else.
I'll revisit this and consider changing them all to the left again.

Reply:
It's just for consistency, if you set `*optional*` to the left, the `*required*` should be there too. If visually is not pleasing, then move them to the right.

Reply:
Tried some variations and went with removing the optionals. It makes no sense marking things optional and required if there's fields that have neither. By definition, fields that are not required should be optional (or wrongly documented).

Reply:
True.